resources/views/entities/pages/abilities/index.blade.php

```diff
@@ -17,6 +17,12 @@
                 <i class="fa-solid fa-redo"></i> <span class="hidden-xs hidden-sm">{{ __('entities/abilities.actions.reset') }}</span>
                 <span class="visible-xs visible-sm">{{ __('crud.actions.reset') }}</span>
             </a>
+            <a href="{{ route('abilities.reorder', $entity) }}" class="btn btn-sm btn-default">
+                <i class="fa-solid fa-sort"></i> 
+                <span class="visible-xs visible-sm">
```

Comment:
Missing the text on desktop, it's just an icon?

Reply:
Just an icon